frontend/src/routes/(voters)/parties/[partyId]/+page.svelte

```diff
@@ -6,21 +6,25 @@
   export let data: PageServerData;
 </script>
 
-<section class="card m-8 w-96 bg-base-100 shadow-xl">
-  <div class="card-body">
-    <h2 class="card-title">{data.party.name}</h2>
-    <h3>{data.party.info}</h3>
-    {#if data.party.nominatedCandidates?.length}
-      <p><strong>{$_('candidates.candidates')}:</strong></p>
-      <ul>
-        {#each data.party.nominatedCandidates as candidate}
-          <li>
-            <a href={`/candidates/${candidate.id}`}>
-              {GetFullNameInOrder(candidate.firstName, candidate.lastName)}
-            </a>
-          </li>
-        {/each}
-      </ul>
-    {/if}
+<div class="flex w-full flex-grow flex-col items-center bg-base-300 lg:p-md lg:pb-0">
+  <div class="w-full max-w-xl flex-grow rounded-t-lg bg-base-100 pb-[3.5rem] lg:shadow-xl">
+    <section class="card">
+      <div class="card-body">
+        <h2 class="card-title">{data.party.name}</h2>
```

Comment:
no `h1` in this page?

Reply:
I'll change that to `h1`. Didn't touch it earlier, bc it was a placeholder page.